pkg/security/serializers/serializers_linux.go

```diff
@@ -1097,7 +1099,12 @@ func newNetworkFlowMonitorSerializer(nm *model.NetworkFlowMonitorEvent, e *model
 }
 
 func newSysCtlEventSerializer(sce *model.SysCtlEvent, _ *model.Event) *SysCtlEventSerializer {
+	snapshot := sysctl.NewSnapshot()
+	relPath := path.Join("sys", sce.Name)
+	snapshot.InsertSnapshotEntry(snapshot.Proc, relPath, sce.Value)
+
 	return &SysCtlEventSerializer{
+		Proc:              snapshot.Proc,
```

Comment:
I'm not sure I understand why we are adding a Snapshot field to the sysctl events, isn't this duplicating the event information into this new field?

Reply:
Yes the goal is to simplify querying / exploring "why" a `sysctl` value has changed on a host by making sure we use the same serialization (and thus the same search query for both events in Datadog). We could remove the `sysctl.name` field in the `sysctl` events, but I'd like to keep it for better visualization in dashboards.